src/server.ts

```diff
@@ -90,34 +102,38 @@ app.use('/api', [
 app.use(metrics([gatsbyRegister, register]))
 
 app.use(sitemap)
-app.use('/', social)
+app.use(IS_NEW_ROLLOUT ? '/governance/' : '/', social)
```

Comment:
I think you could do the same for this file :)

`const routePrefix = IS_NEW_ROLLOUT ? '/governance' : ''`

and use it in

```typescript
app.get('${routePrefix}/balance', ... )
app.get('${routePrefix}/grants', ... )
app.get('${routePrefix}/', withCors(... )
```

You could even declare `routePrefix` in the constants file and import it in both places